enclave-manager/web/packages/app/src/emui/enclaves/components/modals/ConnectEnclaveModal.tsx

```diff
@@ -8,17 +8,16 @@ import {
   ModalOverlay,
 } from "@chakra-ui/react";
 import { FileDisplay } from "kurtosis-ui-components";
+import { apiKey, instanceUUID } from "../../../../cookies";
 import { EnclaveFullInfo } from "../../types";
 
 export type ConnectEnclaveModalProps = {
   enclave: EnclaveFullInfo;
-  instanceUUID: string;
-  apiKey: string;
   isOpen: boolean;
   onClose: () => void;
 };
 
-export const ConnectEnclaveModal = ({ isOpen, onClose, enclave, instanceUUID, apiKey }: ConnectEnclaveModalProps) => {
+export const ConnectEnclaveModal = ({ isOpen, onClose, enclave }: ConnectEnclaveModalProps) => {
```

Comment:
@Dartoxian you may want to coordinate this with @laurentluce . I think he's making changes in this area too and looking to introduce the API key here.